qutebrowser/completion/models/miscmodels.py

```diff
@@ -60,17 +60,31 @@ def helptopic():
 
 def quickmark():
     """A CompletionModel filled with all quickmarks."""
+    def delete(data):
```

Comment:
Coulda sworn I fixed that before I pushed :grin:. I don't see an obvious way to make pylint ignore that though.